dbus/src/blocking/generated_org_freedesktop_dbus.rs

```diff
@@ -116,94 +194,16 @@ impl<'a, T: blocking::BlockingSender, C: ::std::ops::Deref<Target=T>> DBus for b
             .and_then(|r: (String, )| Ok(r.0, ))
     }
 
-    fn get_connection_credentials(&self, arg0: &str) -> Result<::std::collections::HashMap<String, arg::Variant<Box<dyn arg::RefArg + 'static>>>, dbus::Error> {
+    fn get_connection_credentials(&self, arg0: &str) -> Result<arg::PropMap, dbus::Error> {
         self.method_call("org.freedesktop.DBus", "GetConnectionCredentials", (arg0, ))
-            .and_then(|r: (::std::collections::HashMap<String, arg::Variant<Box<dyn arg::RefArg + 'static>>>, )| Ok(r.0, ))
+            .and_then(|r: (arg::PropMap, )| Ok(r.0, ))
     }
 
     fn features(&self) -> Result<Vec<String>, dbus::Error> {
-        <Self as blocking::stdintf::org_freedesktop_dbus::Properties>::get(&self, "org.freedesktop.DBus", "Features")
+        <Self as blocking::stdintf::org_freedesktop_dbus::Properties>::get(self, "org.freedesktop.DBus", "Features")
```

Comment:
Confused about this change, from `&self` to `self`?

Reply:
The self parameter is already a `&self`, so its type is `&Self`, borrowing it again isn't needed

Again, this code is generated by dbus-codegen, so I didn't write that myself